WooCommerce/Classes/ViewRelated/ReusableViews/SwiftUI Components/AdaptiveModalContainer.swift

```diff
@@ -1,3 +1,4 @@
+import Foundation
```

Comment:
I don't think this is needed, SwiftUI gives you Foundation.

Reply:
One of those cases where Xcode thinks is being funny, for some reason didn't allow me to compile without importing Foundation when declaring the `NSLocalizedString`, but now works just fine.

Reply:
Ugh I know that feeling.